### PR TITLE
PSQLADM-467 Release notes for Proxysql 2.4.8

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/2.4.8.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml
modified:   docs/index.md
new file:   variables.yml

### CODE REVIEW

File: docs/2.4.8.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Release highlights
+
+This release includes all security fixes available in ProxySQL 2.4.8. The security fix introduced by ProxySQL and included in this release is:
+
+* OpenSSL was updated to 3.0.8.
+
+For more information on these items and other improvements and bug fixes, see the ProxySQL [2.4.8 release notes](https://github.com/sysown/proxysql/releases/tag/v2.4.8).
+
+ProxySQL Admin and Percona Scheduler admin have not changed since the previous release.
```
Comment (patrickbirch) [inline docs/2.4.8.md:24]: pxc_scheduler_handler have not changed...

Comment (alina-derkach-oaza) [reply docs/2.4.8.md]: updated

File: docs/2.4.8.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+|---|---|
+| Install instructions | [ProxySQL and proxysql-admin](https://docs.percona.com/proxysql/install-v2.html) |
+| Download this version | [ProxySQL 2.x](https://www.percona.com/downloads/proxysql2/) |
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator as a solution for complex replication topology issues. 
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
+
+The [ProxySQL Admin (proxysql-admin)](proxysql-admin-tool-v2-config.md) utility simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](proxysql-v1.md) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](proxysql-admin-tool-functions.md) supports Percona XtraDB Cluster without custom scripts.
```
Comment (patrickbirch) [inline docs/2.4.8.md:12]: tool? utility?

File: docs/2.4.8.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The [ProxySQL 2.x.x and pxc_scheduler_handler](psh-overview.md) tool can automatically perform a failover due to node failures, service degradation, or maintenance. The pxc_scheduler_handler tool has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
```
Comment (patrickbirch) [inline docs/2.4.8.md:14]: tool? utility?

**Review by patrickbirch — APPROVED** [review]
LGTM

File: docs/2.4.8.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/2.4.8.md]: updated

**Review by mohitj1988 — APPROVED** [review]

**Pull request merged**